pkg/printers/template.go

```diff
@@ -31,6 +31,20 @@ type TemplatePrinter struct {
 	template    *template.Template
 }
 
+func NewTemplatePrinterBuilder(flags *TemplatePrintFlags) (ResourcePrinter, bool, error) {
+	if *flags.OutputFormat != "template" && *flags.OutputFormat != "go-template" {
+		return nil, false, nil
+	}
+
+	p, err := NewTemplatePrinter([]byte(*flags.TemplateArgument))
```

Comment:
You can't dereference, this may be nil.  You may need to provide an error in that case, but don't panic.